target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -340,7 +340,7 @@ define Device/spectrum_sax1v1k
 	DEVICE_DTS_CONFIG := config@rt5010w-d187-rev6
 	SOC := ipq8072
 	IMAGES := sysupgrade.bin
-	DEVICE_PACKAGES := ipq-wifi-spectrum_sax1v1k
+	DEVICE_PACKAGES := kmod-fs-ext4 e2fsprogs kmod-fs-f2fs f2fs-tools ipq-wifi-spectrum_sax1v1k
```

Comment:
"kmod-fs-ext4" and "e2fsprogs" are already selected in toplevel Makefile of qualcommax target.

https://github.com/openwrt/openwrt/blob/7850e197670afff38426075953ba6a71f0f431c8/target/linux/qualcommax/Makefile#L19

Reply:
thank you! fixed.

(btw, this is really ugly!)

Reply:
The default EXT4 packages predate the recent support for per-device initramfs images.